### PR TITLE
feat: New theme capr4n

### PR DESCRIPTION
**Pull request by Naccl**
### Prerequisites

- [x] I have read and understood the `CONTRIBUTING` guide
- [x] The commit message follows the [conventional commits][cc] guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes/features)

### Description

This theme is based on aliens and unicorn.

capr4n

### CODE REVIEW

File: themes/capr4n.omp.json [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        },
+        {
+          "type": "path",
+          "style": "powerline",
+          "powerline_symbol": "\uE0B0",
+          "foreground": "ffffff",
+          "background": "#C678DD",
+          "properties": {
+            "max_depth": 2,
+            "prefix": " ",
```
Comment (JanDeDobbeleer) [inline themes/capr4n.omp.json]: this is the default so that can be omitted, same for `postfix` below

File: themes/capr4n.omp.json [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  "console_title_template": "{{if .Root}} ⚡ {{end}}{{.User}} ➔ 📁{{.Folder}}",
+  "blocks": [
+    {
+      "type": "rprompt",
+      "alignment": "right",
+      "segments": [
+        {
+          "type": "time",
+          "style": "powerline",
+          "powerline_symbol": "",
```
Comment (JanDeDobbeleer) [inline themes/capr4n.omp.json]: why Powerline when you're not using a symbol? As that would imply `"plain"`, no

File: themes/capr4n.omp.json [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+          "foreground": "#ffffff",
+          "background": "#003543",
+          "properties": {
+            "prefix": "<#fff> \uf64f </>"
+          }
+        },
+        {
+          "type": "executiontime",
+          "style": "diamond",
```
Comment (JanDeDobbeleer) [inline themes/capr4n.omp.json]: same remark as above, this implies `"plain"`

**Review by JanDeDobbeleer — CHANGES_REQUESTED** [review]

File: themes/capr4n.omp.json (hunk 2 above)
Comment (Naccl) [reply themes/capr4n.omp.json]: i forgot to update this after making changes based on other themes:joy:
i will update them soon

**Pull request merged**